data/borderlines.json

```diff
@@ -79,6 +84,11 @@
           "name": "Himalaya Mahalaxmi Tea Garden",
           "latitude": 26.6539691,
           "longitude": 87.603649
+        },
+        {
+          "name": "Sano Baraghare",
```

Comment:
Sano baraghare is a place in Damak not a landmark

Reply:
My bad, added New Amda Hospital in it's place